### PR TITLE
New UI, Old docs

### PR DESCRIPTION
**Pull request by jordanranz**
*Description of changes:*
- Amplify UI Components documentation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -55,806 +55,810 @@ plugins:
...
+      - title: Amplify iOS
+        excerpt: Version of the docs
+        cta: Read more
+        class: aws-icon-ps-60-circular-circuit
+        url: "/ios/start"
+      - title: iOS SDK 2.0
+        excerpt: Version of the docs
+        cta: Read more
+        class: aws-icon-ps-60-circular-circuit
+        url: "/sdk/ios/start"
```
Comment (sammartinez) [inline _config.yml:619]: Seeing that most of these changes look like prettier doing something, do we want to do the same config change we did for JS last night?

File: _config.yml [hunk 2]
```diff
@@ -55,806 +55,810 @@ plugins:
...
+        item_external_url: "https://aws-amplify.github.io/media/ui_library"
+        id: ui
+    ios:
+      theme: "dark"
+  footer:
+    content:
+      logo:
+        text: AWS Amplify is supported by Amazon Web Services
+        image: true
+      copyright: © 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (jordanranz) [reply _config.yml:91]: sure

File: _config.yml (hunk 1 above)
Comment (jordanranz) [reply _config.yml:619]: There is no prettier in docs v1 so I don't believe so. Not sure why my VS code formatted everything..

Comment (jordanranz) [reply _config.yml:619]: I think this is valid formatting though

Comment (sammartinez) [reply _config.yml:619]: Agreed it was more of a callout if it was something with Prettier, which it isn't, and also just seeing what it was.

Comment (sammartinez) [reply _config.yml:619]: This was non-blocking btw

**Pull request merged**